index.js

```diff
@@ -72,7 +82,7 @@ function deepmergeConstructor (options) {
   }
 
   function isPrimitiveOrBuiltIn (value) {
-    return typeof value !== 'object' || value === null || value instanceof RegExp || value instanceof Date
+    return typeof value !== 'object' || value === null || value instanceof RegExp || value instanceof Date || value instanceof Buffer
```

Comment:
This package potentially also is using in browsers.

I would prefer something more like this:

```javascript
  const isPrimitiveOrBuiltIn = typeof Buffer !== 'undefined'
    ? function isPrimitiveOrBuiltIn(value) {
        return typeof value !== 'object' || value === null || value instanceof RegExp || value instanceof Date || value instanceof Buffer
      }
    : function isPrimitiveOrBuiltIn(value) {
        return typeof value !== 'object' || value === null || value instanceof RegExp || value instanceof Date
      }
```